metricks.gemspec

```diff
@@ -15,5 +15,4 @@ Gem::Specification.new do |gem|
   gem.email         = ['help@krystal.uk']
   gem.required_ruby_version = '>= 2.7'
   gem.add_runtime_dependency 'activerecord', '>= 5.0'
-  gem.add_runtime_dependency 'with_advisory_lock', '>= 4.6', '< 5.0'
```

Comment:
one thing to watch out for when upgrading this gem is that these versions of with_advisory_lock will no longer be enforced. FWIW I ran this gem with version 5.1.0 of with_advisory_lock in my testing and it seemed fine.

[The changelog](https://github.com/ClosureTree/with_advisory_lock/blob/master/CHANGELOG.md) shows it dropped support for ruby below 2.7 and activerecord below 6.1

Ultimately the host app can decide what version of with_advisory_lock they want.